src/Swarm/Game/Failure.hs

```diff
@@ -26,4 +28,5 @@ data LoadingFailure
   = DoesNotExist Entry
   | EntryNot Entry
   | CanNotParse ParseException
+  | Duplicate Text
```

Comment:
Should this `Duplicate` constructor have another field that indicates what kind of thing was duplicated?  E.g. an `Entity`, a `Subworld`, etc.